PlayFabClientSDK/source/PluginContract.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace PlayFab
+{
+    public enum PluginContract
```

Comment:
What is Contract in this context? It seems like this is an enumeration of PlayFabPluginTypes.

Reply:
Each of these enums maps to a specific Interface, which is defined elsewhere.
The enum serves as a dictionary index-key for each interface type.

Reply:
Paul just commented on that.

---
In reply to: [201481143](https://github.com/PlayFab/CSharpSDK/pull/97#discussion_r201481143) [](ancestors = 201481143)